### DISCUSSION TITLE
File names in release are badly truncated

### DISCUSSION THREAD

**Discussion by soredake** [community/community#86702] [Mobile]
### Select Topic Area

Bug

### Body

How GitHub app shows file names (you can't see name and version, unusable):
![Screenshot_20240112-105753_GitHub](https://github.com/community/community/assets/5204968/8e0ace11-8bc6-46ec-8b6e-ef5b9db868f2)

How GitHub web version shows file names (you can see name and version, usable):
![Screenshot_20240112-105735_Firefox Beta](https://github.com/community/community/assets/5204968/35bfcc67-39cf-457a-83f6-5f4347b6af6a)

**Comment by elisealix22** [comment]
Thank you again for the screenshots, @soredake ! I created an issue to track this internally and our Android team will look into it.

**Comment by Amq2705** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> How GitHub app shows file names (you can't see name and version, unusable):
> 
> ![Screenshot_20240112-105753_GitHub](https://github.com/community/community/assets/5204968/8e0ace11-8bc6-46ec-8b6e-ef5b9db868f2)
> 
> 
> 
> 
> 
> How GitHub web version shows file names (you can see name and version, usable):
> 
> ![Screenshot_20240112-105735_Firefox Beta](https://github.com/community/community/assets/5204968/35bfcc67-39cf-457a-83f6-5f4347b6af6a)
> 
>

**Comment by soredake** [comment]
This seems to be fixed now.

![Screenshot_20240524-173938.png](https://github.com/community/community/assets/5204968/9b675571-8053-465c-a463-dd5190375da1)
